jovo-integrations/jovo-db-dynamodb/src/DynamoDb.ts

```diff
@@ -9,6 +9,9 @@ export interface Config extends PluginConfig {
   tableName?: string;
   createTableOnInit?: boolean;
   primaryKeyColumn?: string;
+  primaryKeyPrefix?: string;
```

Comment:
Please sort the properties alphabetically.
https://travis-ci.org/github/jovotech/jovo-framework/jobs/688064719